src/rgw/rgw_rados.cc

```diff
@@ -385,7 +385,9 @@ int RGWSystemMetaObj::init(CephContext *_cct, RGWRados *_store, bool setup_obj,
     } else if (!old_format) {
       r = read_id(name, id);
       if (r < 0) {
-        ldout(cct, 0) << "error in read_id for object name: " << name << " : " << cpp_strerror(-r) << dendl;
+        if (r != -ENOENT) {
```

Comment:
Hi Jos - We generally don't "clean up" master commits when backporting, preferring to take them verbatim unless doing so causes a serious error.